package/cfg/config.yaml

```diff
@@ -207,7 +207,8 @@ version_mapping:
   "rke-1.16": "rke-cis-1.6-permissive"
   "rke-1.17": "rke-cis-1.6-permissive"
   "rke-1.18": "rke-cis-1.6-permissive"
-  "rke-1.19": "rke-cis-1.6-permissive"
+  "rke-1.19": "rke-cis-1.20-permissive"
+  "rke-1.20": "rke-cis-1.20-permissive"
```

Comment:
Quick question, shouldn't we also add the hardened profile `rke-cis-1.20-hardened` here?

Reply:
kube-bench requires either `kubernetes version` OR `benchmark version` when running the test. In our case we run CIS scan through UI only and there we are always passing the benchmark version/profiles (For example `rke-cis-1.6-hardened`) so we dont need to actually update the version_mapping block. @prachidamle Is my understanding correct?

Reply:
This version_mapping block specifies the default benchmark version to be used for a given kubernetes version for the cluster type. We should update it to point to the default profile to be used in scanning, which will be the permissive profile.